### PR TITLE
Updating snippets documentation README.md

### PR DESCRIPTION
**Pull request by gspencergoog**
## Description

Updates the README file for snippets.

### CODE REVIEW

**Review by shihaohong — APPROVED** [review]
LGTM, other than a couple of nits.

I think the README files should also adhere to the 80 char minimum per line rule, since this helps with reading the docs when viewing them in an IDE.

**Comment by shihaohong** [comment]
Also, the analyzer seems to be failing on trailing whitespaces, so that has to be fixed as well

File: dev/snippets/README.md [hunk 1]
```diff
@@ -189,3 +194,19 @@ generating a local copy of the API documentation.
 search bar will not work locally, so open `./dev/docs/doc/index.html` to
 navigate through the documentation, or search `./dev/docs/doc/flutter` for your
 page of interest.
+
+Note that generating the sample output will not allow you to run your code in DartPad, because DartPad pulls the code it runs from the appropriate docs server (master or stable).
+
+Copy your code and paste it into a regular DartPad instance to test if it runs in DartPad.  You can get the code that will be produced by the interpolation of your code into a template by running the sample analysis locally (see the next section), and pasting the output file into a dartpad at https://dartpad.dartlang.org.
```
[the remark this answers is not in the document]
Comment (gspencergoog) [reply dev/snippets/README.md]: Thanks, that's better.

**Comment by gspencergoog** [comment]
I reflowed all the paragraphs, and formatted the lists to be more consistent in a "text" view, and removed the trailing whitespace.

**Pull request merged**